### ISSUE TITLE
embulk v0.8.28 bundle command is not working.

### ISSUE THREAD

**Issue by smdmts** [embulk/embulk#750]
embulk v0.8.28  breaking bundle command. Maybe related to https://github.com/embulk/embulk/issues/748

- 0.8.28
```
embulk bundle
2017-07-28 09:38:46.392 +0000: Embulk v0.8.28
Exception in thread "main" java.lang.NullPointerException
	at org.embulk.cli.EmbulkRun.runSubcommand(EmbulkRun.java:401)
	at org.embulk.cli.EmbulkRun.run(EmbulkRun.java:80)
	at org.embulk.cli.EmbulkBundle.checkBundleWithEmbulkVersion(EmbulkBundle.java:42)
	at org.embulk.cli.EmbulkBundle.checkBundle(EmbulkBundle.java:15)
	at org.embulk.cli.Main.main(Main.java:26)
```

- 0.8.27
```
embulk bundle install
2017-07-28 09:43:55.455 +0000: Embulk v0.8.27
Using embulk-executor-mapreduce 0.3.0
```

**Comment by hiroyuki-sato** [comment]
Thank you for reporting.
It reproduced on my machine.

Reproduce steps.

```
embulk mkbundle bundle
cd bundle
embulk bundle install
```

* Embulk 0.8.28
* OS: macOS 10.12.5

**Comment by dmikurube** [comment]
Thanks for reporting! To be fixed in #752.
